src/plugins/cloudinary/models/video-source/video-source.utils.js

```diff
@@ -28,3 +30,31 @@ export function normalizeFormat(format) {
 
   return res;
 }
+
+const isContainCodec = (value) => Object.values(VIDEO_CODEC).some((codec) => value.includes(codec));
```

Comment:
please make sure there are passing unit tests for the utils

Reply:
You should also add an integration test for the specific case this pr is handling